src/multivariate/mvnormal.jl

```diff
@@ -195,29 +195,23 @@ function MvNormal(μ::AbstractVector, Σ::AbstractPDMat)
     MvNormal(convert(AbstractArray{R}, μ), convert(AbstractArray{R}, Σ))
 end
 
-MvNormal(Σ::AbstractPDMat) = MvNormal(Zeros{eltype(Σ)}(dim(Σ)), Σ)
```

Comment:
@devmotion Was it intentional that you deleted this one or did you expect the `MvNormal(Σ::AbstractVecOrMat{<:Real})` to cover? (It doesn't since the PDMats aren't AsbtractMatrices.)

Reply:
No, I don't remember exactly but I guess I expected it to be covered by the general constructor without mean vector. Unfortunate that I missed this.